dubbo-configcenter/dubbo-configcenter-nacos/src/main/java/org/apache/dubbo/configcenter/support/nacos/NacosDynamicConfiguration.java

```diff
@@ -144,14 +142,10 @@ private void setServerAddr(URL url, Properties properties) {
     }
 
     private static void setProperties(URL url, Properties properties) {
-        putPropertyIfAbsent(url, properties, NACOS_NAMING_LOG_NAME);
```

Comment:
`NACOS_NAMING_LOG_NAME` and `NAMING_LOAD_CACHE_AT_START ` are for NacosNamingService.
So in dynamicCofnig, not need it, so remove it.